### ISSUE TITLE
Executables for non python users

### ISSUE THREAD

**Issue by haverland** [haverland/collectmeterdigits#9]
Users should not must have python installed or experience of python to use the collect application.

For all platforms, but especially Windows should be a executable in release page available.

**Comment by haverland** [comment]
Closed

**Comment by JAudi23** [comment]
Is there a specific version of Python required for Ubuntu to get this collectmeterdigits to work?

```
ERROR: Ignored the following versions that require a different python version: 1.6.2 Requires-Python >=3.7,<3.10; 1.6.3 Requires-Python >=3.7,<3.10; 1.7.0 Requires-Python >=3.7,<3.10; 1.7.0rc1 Requires-Python >=3.7,<3.10; 1.7.0rc2 Requires-Python >=3.7,<3.10; 1.7.1 Requires-Python >=3.7,<3.10
ERROR: Could not find a version that satisfies the requirement tflite-runtime (from collectmeterdigits) (from versions: none)
ERROR: No matching distribution found for tflite-runtime
```

**Comment by haverland** [comment]
> Is there a specific version of Python required for Ubuntu to get this collectmeterdigits to work? I get the below error when trying to install the tensorflow portion.
> 
> ```
> ERROR: Ignored the following versions that require a different python version: 1.6.2 Requires-Python >=3.7,<3.10; 1.6.3 Requires-Python >=3.7,<3.10; 1.7.0 Requires-Python >=3.7,<3.10; 1.7.0rc1 Requires-Python >=3.7,<3.10; 1.7.0rc2 Requires-Python >=3.7,<3.10; 1.7.1 Requires-Python >=3.7,<3.10
> ERROR: Could not find a version that satisfies the requirement tflite-runtime (from collectmeterdigits) (from versions: none)
> ERROR: No matching distribution found for tflite-runtime
> ```

Python3 >= 3.8. 

Did you used pip install ... or the executable?

**Comment by JAudi23** [comment]
As per your documentation when I attempted to run:

`pip install git+https://github.com/haverland/collectmeterdigits`

I get a prompt saying:

`Command 'pip' not found, but can be installed with:`
`sudo apt install python3-pip`

I ran the install for pip.  No errors showed up.

I tried to run the command again:

`pip install git+https://github.com/haverland/collectmeterdigits`

and received this:

![image](https://user-images.githubusercontent.com/107599616/223903107-e2c37b9c-eb6b-430a-9890-dbec2c3f7d8d.png)

I installed git using the command:

`sudo apt install git`

and it installed without any errors.

Next I ran the command to install collectmeterdigits again from GitHub:

`pip install git+https://github.com/haverland/collectmeterdigits`

all seems to install properly, but errors with this at the end:

`ERROR: Could not find a version that satisfies the requirement tflite-runtime (from collectmeterdigits) (from versions: none)`
`ERROR: No matching distribution found for tflite-runtime`

This is where I seem to be stuck.

For reference the version of Python I have installed is: Python 3.10.6
